static_frame/core/node_iter.py

```diff
@@ -189,7 +198,7 @@ def apply_iter_items(self,
 
     @doc_inject(selector='apply')
     def apply_iter(self,
-            func: AnyCallable
+            func: TCallableAny
```

Comment:
I think you could constrain this a bit more. Perhaps:
```python
func: TCallableOneArg | TCallableTwoArgs
```

Reply:
This seems desirable, but leads to other issues, so I am going to defer this one for now.